docs/core/diagnostics/runtime-events.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Runtime Events"
+description: Review events emitted by CoreCLR runtime.
+ms.date: "09/14/2020"
+helpviewer_keywords: 
+  - "loader events [.NET Core]"
+  - "ETW, EventPipe loader events (CoreCLR)"
+---
+# .NET Core runtime events
```

Comment:
All of the new files will also need to go into the fundamentals table of contents file, somewhere around [here](https://github.com/dotnet/docs/blob/666327906c8c0b0e969589d422c54f09aa70d6ec/docs/fundamentals/toc.yml#L330) I would think. Also, instead of putting these files under docs/core/, can you put them under docs/fundamentals/? We are slowing moving things out of the "core" folder.

Reply:
Is it an issue that this specifically is documenting the .NET Core subset of events?

Reply:
Yeah I was wondering the same thing as @sdmaclea.

Reply:
@BillWagner to confirm, but I think the idea is to have ".NET" and ".NET Framework", where .NET basically means .NET Core and .NET 5 and later.